fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/migration/TenantDatabaseUpgradeService.java

```diff
@@ -59,6 +60,8 @@ public class TenantDatabaseUpgradeService implements InitializingBean {
     private final TenantDataSourceFactory tenantDataSourceFactory;
     private final Environment environment;
 
+    private final List<CustomTaskChange> customTaskChangesForDependencyInjection;
```

Comment:
Is this needed to ensure the ordering of bean creation? If so, can you make a comment here on that so nobody will delete it?